config/locales/cy.yml

```diff
@@ -567,3 +567,16 @@ cy:
           - text: 'If you need urgent help and you’re in Northern Ireland call Lifeline on <a href="tel:+448088088000" class="govuk-link">0808 808 8000</a>'
           - text: 'If you need urgent help and you’re in Wales call <a href="tel:+44800132737" class="govuk-link">0800 132 737</a> or text ‘help’ to 81066'
           - text: "If it’s an emergency, call 999"
+  footer:
+    links: |
+      <li class="menu__item"><a href="https://llyw.cymru/cysylltu-ni">Cysylltu â ni</a></li>
```

Comment:
We should minimise the amount of HTML in the locale files.

For having a separate href value for each text link, you can see how this is being done already in the same file:
```
items:
          - text: "Find information about your rights as a tenant"
            href: "https://www.gov.uk/government/publications/covid-19-and-renting-guidance-for-landlords-tenants-and-local-authorities"
```

Reply:
This has been done now @nicrodgers